docs/UserGuide.md

```diff
@@ -283,10 +279,11 @@ Figure 5.2.1.1. Example of create budget command
 
 You can use this command to **delete an existing budget**.
 
-When a budget is deleted it will be removed from NUSave, and the 
-deletion will be reflected immediately in the list of budget on the main page of NUSave.
+When a budget is deleted, you will see the budget being removed from the current page.
```

Comment:
Should it be "When you delete a budget, the list view should reflect the changes by removing its card."?